viewing_party/main.py

```diff
@@ -0,0 +1,129 @@
+
+#the functions below are for wave_01
+# 
+
+def create_movie(movie_title,  genre, rating):
+    if not movie_title or not genre or not rating:
+        return None
+    return {
+        "title":  movie_title, 
+        "genre" : genre,
+        "rating" :rating 
+        }
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+  
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie):
+    watched_movie = {}
+    watchlist = user_data["watchlist"]
+    for index in range(len(watchlist)):
+        if watchlist[index]["title"] == movie:
+            watched_movie = watchlist[index]
+            watchlist.pop(index)
+            return add_to_watched(user_data, watched_movie)
+    return user_data
+
+#the functions below are for wave_02 
+def get_watched_avg_rating (user_data):
+    sum = 0 
+    watched = user_data["watched"]
+    for index in range(len(watched)):
+       sum += watched[index]["rating"]
+    if sum: return sum / len(watched)
+    return 0
```

Comment:
what happens if `len(watched)` is zero? maybe instead, check `if watched`, then initiate the for loop.

```suggestion
    sum = 0 
    watched = user_data["watched"]
    if watched:
       for movie in watched:
          sum += watched["rating"]
        sum = sum / len(watched)
     return sum
```

Reply:
This is a great suggestion for readability purposes and the use of watched. Thank you, Claire!